vyper/codegen/module.py

```diff
@@ -91,6 +92,11 @@ def _is_internal(func_ast):
 def _is_payable(func_ast):
     return func_ast._metadata["type"].mutability == StateMutability.PAYABLE
 
+def _is_zero_selector_func(f):
+    abi_sig = f._metadata["signature"].abi_signature_for_kwargs([])
+    selector = abi_method_id(abi_sig)
+    if selector.to_bytes(4, 'big') == b"\x00"*4:
```

Comment:
bit weird, why not `return selector.to_bytes(4, "big") == b"\x00"*4`

or, `return selector == 0`?